PictureApp/Controller/FilterViewController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  FilterViewController.swift
+//  PictureApp
+//
+//  Created by Irina Gorbachenko on 07.12.2021.
+//
+
+import UIKit
+import NetworkLayer
+
+protocol FilterViewControllerDelegate : AnyObject {
+    
+    func filterViewController (_ controller: FilterViewController, didSelectCategory selectedCategory : [String])
```

Comment:
Why do we need 
`_ controller: FilterViewController`

here?

Reply:
Done